Evaluator/Strategies/mixed_strategies_evaluator/tests/test_simple_strategy_evaluator.py

```diff
@@ -61,7 +61,7 @@ async def test_flat_markets(self):
         # market: -12.732688011913623
         # market: -34.64150943396227
         await self.run_test_flat_markets(decimal.Decimal(str(2.429)), decimal.Decimal(str(8.981)),
-                                         decimal.Decimal(str(-5.819)), decimal.Decimal(str(-18.693)))
+                                         decimal.Decimal(str(-6.707)), decimal.Decimal(str(-18.693)))
```

Comment:
do you know where this comes from ?

Reply:
I have no idea. I was thinking about the accuracy improvement due to Asset's Decimal usage but that should have changed most of the values ​​I guess.

Reply:
Yep, thoses changes are surprisingly large